src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -3500,14 +3500,14 @@ internal Collection<PSObject> NewItem(string[] paths, string name, string type,
                     {
                         if (content == null)
                         {
-                            throw PSTraceSource.NewArgumentNullException(SessionStateStrings.NewItemValueNotSpecified, path);
+                            throw PSTraceSource.NewArgumentNullException(nameof(content), SessionStateStrings.NewItemValueNotSpecified, path);
```

Comment:
Please update the resource string. It is used only in the context so we can get better message like:
```
New-Item: Item '.' cannot be created because Target was not specified.
```